django_airavata/apps/workspace/models.py

```diff
@@ -1,2 +1,14 @@
 
 # Create your models here.
+
+from django.db import models
+
+class User_Files(models.Model):
```

Comment:
This looks good but I think you are missing a migration file. Did you run `python manage.py makemigrations`?